src/app/mail/mail-settings/mail-autoresponder/mail-autoresponder.component.scss

```diff
@@ -1,3 +1,8 @@
 ::ng-deep .ql-editor.ql-blank::before {
   color: var(--input-color-placeholder);
 }
+
+.autores-timezone {
+  margin: 0;
+  font-weight: bold;
```

Comment:
This is basic styles, so it can be done by using bootstrap class.
Remove this and add the following class to the element
`font-weight-bold, m-0`